arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/utils.kt

```diff
@@ -8,3 +8,4 @@ public fun <T : Any> Predicate<T>.mapNullable(): (T?) -> Boolean = { t -> t?.let
 
 public const val DeprecatedUnsafeAccess: String = "This function is unsafe and will be removed in future versions of Arrow. Replace or import `arrow.syntax.unsafe.*` if you wish to continue using it in this way"
 public const val DeprecatedAmbiguity: String = "This function is ambiguous and will be removed in future versions of Arrow"
+public const val DeprecatedWiden: String = "This function is useless because Kotlin has variance, and thus will be removed in future versions of Arrow"
```

Comment:
```suggestion
public const val DeprecatedWiden: String = "This function is no longer required because of Kotlin's built-in variance and will be removed in future versions of Arrow"
```

Just trying to avoid the world "useless" (this may give the impression that the developer did something stupid back in the day)